actions/notify-slack/message.json.j2

```diff
@@ -0,0 +1,82 @@
+{
+  "blocks": [
+    {
+      "type": "header",
+      "text": {
+        "type": "plain_text",
+        "emoji": true,
+        "text": ":{{ env("EMOJI") }}: {{ env("TYPE") | title }} {{ env("STATUS") }} for {{ env("APPLICATION") }}"
+      }
+    },
+    {
+      "type": "divider"
+    },
+    {
+      "type": "section",
+      "text": {
+        "type": "mrkdwn",
+        "text": "{{ env("MESSAGE") }}"
+      }
+    },
+    {
+      "type": "section",
+      "fields": [
+        {
+          "type": "mrkdwn",
+          "text": "*Repository*"
+        },
+        {
+          "type": "mrkdwn",
+          "text": "*Github Ref*"
+        },
+        {
+          "type": "mrkdwn",
+          "text": "<https://github.com/{{ env("REPOSITORY") }}|{{ env("REPOSITORY") }}>"
+        },
+        {
+          "type": "mrkdwn",
+          "text": "<https://github.com/{{ env("REPOSITORY") }}/tree/{{ env("REF") }}|{{ env("REF") }}>"
```

Comment:
```suggestion
          "text": "<https://github.com/{{ env("REPOSITORY") }}/tree/{{ env("VERSION") }}|{{ env("VERSION") }}>"
```

Sorry, noticed one more important change which is to use the VERSION env var instead of REF.